.travis.yml

```diff
@@ -63,4 +64,6 @@ script:
   - make
   - make check
   - make deb
+  - docker build -f docker/Dockerfile -t centos:7 .
+  - docker run --rm -v $PWD:/home centos:7 make docker-rpm
```

Comment:
Sounds like this could be hugely simplified if you used a [build matrix](https://docs.travis-ci.com/user/customizing-the-build#Build-Matrix).